testkit/src/main/java/org/apache/calcite/test/QuidemTest.java

```diff
@@ -220,7 +226,8 @@ protected void checkRun(String path) throws Exception {
       // outFile = "/home/fred/calcite/core/build/quidem/test/sql/agg.iq"
       final URL inUrl = QuidemTest.class.getResource("/" + n2u(path));
       inFile = Sources.of(requireNonNull(inUrl, "inUrl")).file();
-      outFile = replaceDir(inFile, "resources", "quidem");
+      outFile = replaceDir(inFile, "resources", "quidem/"
```

Comment:
The previous error was caused by two tests using the same comparison file.

Reply:
so can we do the overloading trick?